app/assets/stylesheets/pictures.scss

```diff
@@ -0,0 +1,112 @@
+// Place all the styles related to the pictures controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
+
+a:link    {color:#000;}
+a:visited {color:#000;}
+a:hover   {color:#000;}
+a:active  {color:#000;}
+
+body {
+  background-color: #F0F0F0;
```

Comment:
Color literals like `#F0F0F0` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#F0F0F0` should be written as `#f0f0f0`